documentation/12-configuration/README.md

```diff
@@ -0,0 +1,270 @@
+```
+
+Logging details here do not affect how the Docker daemon (or other container runtime) handles logging. [See this blog post for a narrative and examples of how to manage log output from the container](https://www.joyent.com/blog/docker-log-drivers).
+
+### `telemetry`
+
+If a `telemetry` option is provided, ContainerPilot will expose a [Prometheus](http://prometheus.io) HTTP client interface that can be used to scrape performance telemetry. The telemetry interface is advertised as a service to the discovery service similar to services configured via the `services` block. Each `sensor` for the telemetry service will run periodically and record values in the [Prometheus client library](https://github.com/prometheus/client_golang). A Prometheus server can then make HTTP requests to the telemetry endpoint.
+
+[Read more](/containerpilot/docs/telemetry).
+
+### `tasks`
+
+Tasks are commands that are run periodically. They are typically used to perform housekeeping such as incremental back-ups, or pushing metrics to systems that cannot collect metrics through service discovery like Prometheus.
+
+[Read more](/containerpilot/docs/tasks).
+
+### Lifecycle fields
+
+- `preStart`, `preStop`, `postStop` and other fields specific to events in the application's lifecycle [have their own section in the docs](/containerpilot/docs/lifecycle).
+- `stopTimeout` Optional amount of time in seconds to wait before killing the application. (defaults to `5`). Providing `-1` will kill the application immediately.
+
+### `interfaces`
+
+The `interfaces` parameter allows for one or more specifications to be used when searching for the advertised IP. The first specification that matches stops the search process, so they should be ordered from most specific to least specific.
+
+- `eth0` : Match the first IPv4 address on `eth0` (alias for `eth0:inet`)
+- `eth0:inet6` : Match the first IPv6 address on `eth0`
+- `eth0[1]` : Match the 2nd IP address on `eth0` (zero-based index)
+- `10.0.0.0/16` : Match the first IP that is contained within the IP Network
+- `fdc6:238c:c4bc::/48` : Match the first IP that is contained within the IPv6 Network
+- `inet` : Match the first IPv4 Address (excluding `127.0.0.0/8`)
+- `inet6` : Match the first IPv6 Address (excluding `::1/128`)
+- `static:192.168.1.100` : Use this Address. Useful for all cases where the IP is not visible in the container
+
+Interfaces and their IP addresses are ordered alphabetically by interface name, then by IP address (lexicographically by bytes).
+
+**Sample ordering**
+
+- `eth0 10.2.0.1 192.168.1.100`
+- `eth1 10.0.0.100 10.0.0.200`
+- `eth2 10.1.0.200 fdc6:238c:c4bc::1`
+- `lo ::1 127.0.0.1`
+
+### Commands & arguments
+
+All executable fields, such as `health`, `preStart`, and `onChange`, accept both a string or an array. If a string is given, the command and its arguments are separated by spaces; otherwise, the first element of the array is the command path, and the rest are its arguments.
```

Comment:
Let's call out which ones specifically: `services/health`, `preStart`, `preStop`, `postStop`, `backends/onChange`, `task/command`, and `telemetry/sensors/check`.